google/cloud/storage/oauth2/compute_engine_credentials.cc

```diff
@@ -13,36 +13,20 @@
 // limitations under the License.
 
 #include "google/cloud/storage/oauth2/compute_engine_credentials.h"
+#include "google/cloud/internal/oauth2_compute_engine_credentials.h"
 #include <nlohmann/json.hpp>
 
 namespace google {
 namespace cloud {
 namespace storage {
 GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
 namespace oauth2 {
+
 StatusOr<ServiceAccountMetadata> ParseMetadataServerResponse(
     storage::internal::HttpResponse const& response) {
```

Comment:
so the common REST library has code like:
https://github.com/googleapis/google-cloud-cpp/blob/4ab2d009a738726dcde670b7e6d40f3480d8528c/google/cloud/internal/oauth2_compute_engine_credentials.cc#L32-L33

Has this sort of processing already happened?

Reply:
Yes.